modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/js/GlobalMenu.js

```diff
@@ -123,43 +123,53 @@ const AppsPanel = ({
 						key={`tabPane-${index}`}
 					>
 						<ClayLayout.Row className="global-apps-nav">
-							{childCategories.map(({key, label, panelApps}) => (
-								<ClayLayout.Col key={key} md>
-									<ul className="list-unstyled">
-										<li className="global-apps-nav-header">
-											{label}
-										</li>
+							<div className="c-p-0 col-md">
```

Comment:
I'm not very familiar with `<ClayLayout>`, maybe this can't be achieved using it, but it feels like using `<div>` is going in the opposite direction of what we've been doing lately.

Reply:
I don't think we have that right now, since it's not a `container`... 🤔

Reply:
I think it would be the equivalent of `<ClayLayout.Col className="c-p-0" md />`.

Reply:
True!! Feel free to push it and send forward or do as you both please with this :D 

I forgot to say Marcos is also on PTO, so this is yours now :D

Reply:
😳